libsqlite3-sys/src/wasm32_unknown_unknown.rs

```diff
@@ -0,0 +1,181 @@
+#[cfg(not(feature = "bundled"))]
+compile_error!("wasm32-unknown-unknown must be built with '--features bundled'");
+
+use crate::{sqlite3_file, sqlite3_vfs, sqlite3_vfs_register, SQLITE_IOERR, SQLITE_OK};
+use std::os::raw::{c_char, c_int, c_void};
+use std::ptr::null_mut;
+
+#[no_mangle]
+pub unsafe extern "C" fn sqlite3_os_init() -> c_int {
+    let vfs = sqlite3_vfs {
+        iVersion: 1,
+        szOsFile: 0,
+        mxPathname: 1024,
+        pNext: null_mut(),
+        zName: "libsqlite3-sys\0".as_ptr() as *const c_char,
+        pAppData: null_mut(),
+        xOpen: Some(wasm_vfs_open),
+        xDelete: Some(wasm_vfs_delete),
+        xAccess: Some(wasm_vfs_access),
+        xFullPathname: Some(wasm_vfs_fullpathname),
+        xDlOpen: Some(wasm_vfs_dlopen),
+        xDlError: Some(wasm_vfs_dlerror),
+        xDlSym: Some(wasm_vfs_dlsym),
+        xDlClose: Some(wasm_vfs_dlclose),
+        xRandomness: Some(wasm_vfs_randomness),
+        xSleep: Some(wasm_vfs_sleep),
+        xCurrentTime: Some(wasm_vfs_currenttime),
+        xGetLastError: None,
+        xCurrentTimeInt64: None,
+        xSetSystemCall: None,
+        xGetSystemCall: None,
+        xNextSystemCall: None,
+    };
+
+    sqlite3_vfs_register(Box::leak(Box::new(vfs)), 1)
+}
+
+const fn max(a: usize, b: usize) -> usize {
+    [a, b][(a < b) as usize]
+}
+
+const ALIGN: usize = max(
```

Comment:
This is subtly wrong (too low) for targets other than wasm32-unknown-unknown (in particular, 64 bit targets).

For example, I could imagine this code not getting updated if wasm64-unknown-unknown support is added.

It's probably better to use 16 instead of 8 for the hard-coded alignment bound, which is a somewhat safer value to hard-code (it's a *very* common `max_align_t` value for 64bit targets).

Also add a comment about this, please.